frontend/src/components/StyledTextField/StyledTextField.style.ts

```diff
@@ -0,0 +1,20 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+export default makeStyles((theme) => ({
+  cssLabel: {
+    color : '#FFF !important'
+  },
+
+  cssFocused: { },
```

Comment:
seems unnecessary

Reply:
I know it looks weird, but it has to stay that way otherwise styles in material-ui don't work.